torch/onnx/_internal/exporter/_registration.py

```diff
@@ -274,5 +278,19 @@ def is_registered(self, target: TorchOp) -> bool:
         """
         return bool(self.get_decomps(target))
 
+    def cleanup_registry(self) -> None:
```

Comment:
Could you explain a bit one why the clean up is needed?

Reply:
Added more info to function docstring